lib/src/main/java/com/nextcloud/android/sso/model/FilesAppType.java

```diff
@@ -0,0 +1,33 @@
+package com.nextcloud.android.sso.model;
```

Comment:
Please add licence

Reply:
Should i also add it to other files like
- `AidlNetworkRequest`
- `VersionCheckHelper`
- `UiExceptionManager`
- and various other existing files where it is not present?

Reply:
I added the license header to the files i saw without, a few of them have them on a different place, that's why i missed it....